app/routes/popup/Transaction/components/Layout.js

```diff
@@ -39,13 +39,12 @@ class Layout extends Component {
     } = this.props
 
     BigNumber.config({ ROUNDING_MODE: BigNumber.ROUND_UP })
-    const transactionValue = value ? toGWei(new BigNumber(value)) : new BigNumber(0)
     const totalGas = estimations && new BigNumber(estimations.dataGas).plus(new BigNumber(estimations.safeTxGas))
     const transactionFee = totalGas && toGWei(totalGas.times(new BigNumber(-estimations.gasPrice)))
```

Comment:
Have you checked how those methods work if they receive not common values? Like extremely large ones, undefined, nulls?

Any test for understanding the flow?